lib/Hero/Carousel.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+import { AiOutlineLeft, AiOutlineRight } from "react-icons/ai";
+import Swipe from "react-easy-swipe";
+
+const Carousel = ({children}) => {
```

Comment:
Leave spaces in your object destructuring notation: `const Carousel = ({ children }) => {`